docs/conceptual/setting-cus.rst

```diff
@@ -0,0 +1,47 @@
+.. meta::
+    :description: Setting the number of CUs
+    :keywords: AMD, ROCm, cu, number of cus
+
+.. _env-variables-reference:
+
+*************************************************************
+Setting the number of CUs
+*************************************************************
+
+When using GPUs to accelerate compute workloads, it sometimes becomes necessary
+to configure the hardware's usage of Compute Units (CU). This is a more advanced
```

Comment:
Perhaps we need to call this out as Tip.
If experimentation can cause bad things to happen like damaging the hardware, we should put in a warning.

Reply:
Fixed.